src/admin-views/tribe-options-help.php

```diff
@@ -4,97 +4,70 @@
-$support_text[] = array(
-	'type' => 'ol',
-	sprintf( esc_html__( '%sTemplate tags, functions, and hooks & filters%s for The Events Calendar &amp; Events Calendar PRO', 'tribe-common' ), '<a href="http://m.tri.be/fk" target="blank">', '</a>' ),
-	sprintf( esc_html__( '%sFrequently Asked Questions%s ranging from the most basic setup questions to advanced themer tweaks', 'tribe-common' ), '<a href="' . $help->get_ga_link( 'knowledgebase' ) . '" target="blank">', '</a>' ),
-	sprintf( esc_html__( '%sTutorials%s written by both members of our team and users from the community, covering custom queries, integration with third-party themes and plugins, etc.', 'tribe-common' ), '<a href="' . $help->get_ga_link( 'the-events-calendar-for-wordpress-tutorials' ) . '" target="blank">', '</a>' ),
-	esc_html__( "Release notes for painting an overall picture of the plugin's lifecycle and when features/bug fixes were introduced.", 'tribe-common' ),
-	sprintf( esc_html__( "%sAdd-on documentation%s for all of Modern Tribe's official extensions for The Events Calendar (including WooTickets, Community Events, Eventbrite Tickets, Facebook Events, etc)", 'tribe-common' ), '<a href="' . $help->get_ga_link( 'knowledgebase-category/primers' ) . '" target="blank">', '</a>' ),
+$sections = array(
+	array(
+		'type' => 'box',
+		'id' => 'feature-box',
+		'priority' => 0,
+		'content' => array(
+			'<img src="' . esc_url( plugins_url( 'resources/images/modern-tribe@2x.png', dirname( __FILE__ ) ) ) . '" alt="Modern Tribe Inc." title="Modern Tribe Inc.">',
+			sprintf( esc_html__( 'Thanks you for using %s! All of us at Modern Tribe sincerely appreciate it and are stoked to have you on board.', 'tribe-common' ), $help->get_plugins_text() ),
+		),
+	),
+
+	array(
+		'id' => 'support',
+		'title' => __( 'Getting Support', 'tribe-common' ),
+		'priority' => 10,
+		'content' => array(
+			0 => sprintf( __( 'The %s on our website is a great place to find tips and tricks for customizing the plugins and its functionality. It is an extensive collection of tested and updated methods to help you use, tweak, and customize the front-end in creative ways.', 'tribe-common' ), '<a href="http://m.tri.be/18j9" target="_blank">' . __( 'Knowledgebase', 'tribe-common' ) . '</a>' ),
+
+			50 => sprintf( __( '<strong>Want to dive deeper?</strong> Check out our %s for developers.', 'tribe-common' ), '<a href="http://m.tri.be/18jf" target="_blank">' . __( 'list of available functions', 'tribe-common' ) . '</a>' ),
+		),
+	),
+
+	array(
+		'id' => 'extra-help',
+		'title' => __( 'I Need Extra Help!', 'tribe-common' ),
+		'priority' => 20,
+		'content' => array(
+			0 => __( 'While the resources above help solve a majority of the issues we see, there are times you might be looking for extra support. If you need assistance with a legitimate bug with the plugin and would like us to take a look please follow these steps:', 'tribe-common' ),
+
+			10 => array(
```

Comment:
Why use numeric indexes - it seems like it could be somewhat fragile if/when this content changes? Is it to avoid the need for `insert_after_array_index()`?